lib/account/widgets/editable_row.dart

```diff
@@ -33,7 +33,10 @@ class EditableRow extends StatelessWidget {
                   child: IconButton(
                     tooltip: S.of(context).edit,
                     icon: const Icon(Icons.edit),
-                    onPressed: onPressed,
+                    onPressed: () {
```

Comment:
what's the point of changing `onPressed`? It worked fine the way it was (and it was shorter too)

Reply:
my mistake, didn't change it back after debuging